### ISSUE TITLE
Duplicate entry for term "bug" in infotips.json

### ISSUE THREAD

**Issue by jarifibrahim** [fabric8io/fabric8-online-docs#288]
```
{
   "d6c3bc51-f623-4aa4-bea4-4e1d68a27661": {
      "term": "backlog",
      "en_EN": "A backlog is a list of work items that have not been triaged and assigned to a specific iteration."
   }
},
{
   "e8d54bf3-f89e-46e5-86f7-4af6475863b0": {
      "term": "bug",
      "en_EN": "Defect that causes unexpected behavior in the software."
   }
},
{
   "1694e637-2f9b-40ec-8fa8-a22472850ff9": {
      "term": "companinon dependency",
      "en_EN": "Based on the analysis of your stack, {osio} suggests additional dependencies that can add value to your stack."
   }
},
{
   "23c322f1-53b1-4286-b524-37ab58124823": {
      "term": "experience",
      "en_EN": "Experience describes the envisioned user experience in the product to actualize a parent work item. Each parent work item can have multiple experiences."
   }
},
{
   "dbfc82e1-5a40-44bd-af70-f2dc1a1927a4": {
      "term": "feature",
      "en_EN": "A feature is a detailed user-story that helps actualize parent experiences. It can support multiple experiences and is generally achievable within a sprint."
   }
},
{
   "e8d54bf3-f89e-46e5-86f7-4af6475863b0": {
      "term": "bug",
      "en_EN": "Defect that causes unexpected behavior in the software."
   }
},
```
Please see https://docs.prod-preview.openshift.io/json/infotips.json

**Comment by Preeticp** [comment]
Thanks @jarifibrahim, the duplicate reference has been removed from the User Guide glossary, by PR #260. The infotips.json will be updated with these as well as the changes brought in through PR #282.

**Comment by jarifibrahim** [comment]
Thanks @Preeticp

**Comment by Preeticp** [comment]
This has been fixed via #301
